rotkehlchen/data_import/importers/binance.py

```diff
@@ -773,8 +774,9 @@ def _process_multiple_binance_entries(
         """Processes binance entries that are represented with 2+ rows in a csv file.
         Returns Entry type and entries count if any entries were processed. Otherwise, None and 0.
         """
+        operations = [row['Operation'] for row in rows]
         for multiple_entry_class in MULTIPLE_BINANCE_ENTRIES:
-            if multiple_entry_class.are_entries([row['Operation'] for row in rows]):
+            if multiple_entry_class.are_entries(operations):
```

Comment:
we were creating the list multiple times here right?

Reply:
Yes, it was, but it was unnecessary since `rows` wasn't changing at all. This way it only creates the list once, plus the list is available to put in the error message.